eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Usage: create_fasta_from_assembly_report.sh </path/to/assembly_report.txt>
+
+if [ "$#" -lt 3 ]
```

Comment:
Not critical, but I usually check it's not equal to the expected number. An extra word between the first parameters will silently blow everything up. I think 
```suggestion
if [ "$#" -ne 3 ]
```
should work